.github/workflows/ruby.yml

```diff
@@ -0,0 +1,23 @@
+name: CI
+
+on:
+  pull_request:
+  push:
+    branches:
+      - 'master'
+
+concurrency:
+  group: ${{ github.ref_name }}-${{ github.workflow }}
+  cancel-in-progress: true
+
+jobs:
+  rubocop:
+    name: Rubocop
+    uses: theforeman/actions/.github/workflows/rubocop.yml@v0
+    with:
+        command: bundle exec rubocop --parallel --format github
+
+  test:
+    name: Ruby
+    needs: rubocop
+    uses: theforeman/actions/.github/workflows/foreman_plugin.yml@v0
```

Comment:
```suggestion
    name: Tests
    needs: rubocop
    uses: theforeman/actions/.github/workflows/test-gem.yml@v0
    with:
      command: bundle exec rake test
```

Reply:
hammer_cli_foreman_bootdisk is a plugin as per https://projects.theforeman.org/projects/hammer-cli/wiki/List_of_Plugins

Reply:
I think you can also remove the `needs: rubocop` here.